docs/GettingStarted.rst

```diff
@@ -1184,8 +1184,39 @@ The article is shared into the space's activity stream.
 
 
 .. note:: The |image157| button is grey and unclickable until the mandatory fields *title* and *content* are filled. 
+
          
-		  
+Sharing a news from full form
+----------------------------------
+
+To share an article from the full form, you have to display the small form as explained in the previous paragraph and click on "More options" button.
+
+|image159|
+
+The full form contains :
+
+- A **Title** field :  Allows you to enter the news title. The title should not exceed 150 characters: Beyond that limit you will not be allowed to write. This field is mandatory.
+
+- A **Summary** field : Allows you to enter the news summary. It is optional and will be inserted after the title in the news preview. 
```

Comment:
We should mention that in case the summary is not filled, the news preview displays the content.